### ISSUE TITLE
Customize / change behavior - push notification

### ISSUE THREAD

**Issue by vcanovas** [chat-sdk/chat-sdk-android#300]
Hello,
I'm working with notifications and all working fine. I would like to change the icon, add extra text or hide notification when if needed.

I 'm seeing the chat-sdk-firebase-push module, "DefaultBroadcastReceiver", createMessageNotification method... but always show the same notification format. 

How can I modify the behaviour?

Finally, if I click on the notification does not launch the app.

**Comment by ramonpoca** [comment]
When the App is in the  background and receives a new message, the behaviour is different amongst devices with version 8+ and devices with versions below (5, 6, 7).
We've  implemented a custom `BaseInterfaceAdapter` to customize the action when user clicks on the notification (`public Class getChatActivity()`).

On older versions, we get two notifications (see images). One opens the app, the other doesn't.
On 8+ we get a single notification, with no actions and no icon, which does not wake up the App.

In the manifest for `chat-sdk-firebase-push` lib, we see GCM usage, instead of FCM.
We don't know if this is the reason of the different behavior.

Our setup:

```config
compileSdkVersion 27
minSdkVersion 17
targetSdkVersion 27

DEPENDENCIES:
compile 'co.chatsdk.chatsdk:chat-sdk-core:4.1.7'
compile 'co.chatsdk.chatsdk:chat-sdk-ui:4.1.7'
compile 'co.chatsdk.chatsdk:chat-sdk-firebase-adapter:4.1.7'
compile 'co.chatsdk.chatsdk:chat-sdk-firebase-file-storage:4.1.7'
compile 'co.chatsdk.chatsdk:chat-sdk-firebase-ui:4.1.7'
compile 'co.chatsdk.chatsdk:chat-sdk-firebase-push:4.1.7'
```
(NB: We are in this with @vcanovas)

**Comment by bensmiley** [comment]
Fixed as of `4.3.7`.
